### PR TITLE
[Injimob 2265] show verifier name in VP sharing screen and consent screen and add translations for same

### PR DESCRIPTION
**Pull request by PuBHARGAVI**
## Description

> Changed the verifier title in VP sharing screen from Requester to Verification Entity and made changes to show the received verifier's name in consent screen.
> Added translations for the same in all languages.

## Issue ticket number and link

> [INJIMOB-2265](https://mosip.atlassian.net/browse/INJIMOB-2265)

[INJIMOB-2265]: https://mosip.atlassian.net/browse/INJIMOB-2265?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Alka1703 — APPROVED** [review]
Waiting for some more changes related to encoding on swift side.

File: machines/openID4VP/openID4VPSelectors.ts [hunk 1]
```diff
@@ -106,7 +106,10 @@ export function selectIsFaceVerifiedInVPSharing(state: State) {
 }
 
 export function selectVerifierNameInVPSharing(state: State) {
-  return state.context.authenticationResponse['client_metadata']?.name;
+  return (
+    state.context.authenticationResponse['client_metadata']?.['client_name'] ??
```
Comment (sree96) [inline machines/openID4VP/openID4VPSelectors.ts:110]: client_name is this from spec?

**Review by sree96 — CHANGES_REQUESTED** [review]

File: machines/openID4VP/openID4VPSelectors.ts (hunk 1 above)
Comment (PuBHARGAVI) [reply machines/openID4VP/openID4VPSelectors.ts:110]: yes sreenadh
it is mentioned in the spec -> https://www.rfc-editor.org/rfc/rfc7591.txt 
please refer to page 9

Comment (sree96) [reply machines/openID4VP/openID4VPSelectors.ts:110]: https://www.rfc-editor.org/rfc/rfc7591#section-2

**Review by sree96 — APPROVED** [review]

**Pull request merged**